Ultimaker/Cura
# Stop postprocessing plugin creating unnecessary script folders

# Description




# How Has This Been Tested?



- install any plugin that includes i18n functionality. 
It needs to register the parent of its i18n folder to be able to load translations
- remove the `scripts` folder from the configuration folder
- run Cura
- inspect the plugin folder
- a `scripts` folder should _not_ be created there
- a `scripts` folder should have been created in the configuration folder

**Test Configuration**:
* Operating System:
Tested on Windows. Note that on Linux the scripts folder may be created in two locations (preference folder and configuration folder can be split). 

# Checklist:


- [x] My code follows the style guidelines of this project as described in [UltiMaker Meta](https://github.com/Ultimaker/Meta) and [Cura QML best practices](https://github.com/Ultimaker/Cura/wiki/QML-Best-Practices)
- [x] I have read the [Contribution guide](https://github.com/Ultimaker/Cura/blob/main/contributing.md) 
- [x] I have uploaded any files required to test this change
